packages/pyright-internal/src/server.ts

```diff
@@ -203,15 +206,27 @@ function reporterFactory(connection: ProgressReporterConnection): ProgressReport
         },
 
         begin(): void {
-            connection.sendNotification('pyright/beginProgress');
+            _workDoneProgress = connection.window.createWorkDoneProgress();
```

Comment:
This variable would be better served as a variable defined inside of this function. It's not accessed anywhere else.

Reply:
"This function" meaning reporterFactory, I mean.

Reply:
Good point. I've updated the code accordingly.